container/image.bzl

```diff
@@ -217,7 +217,7 @@ def _repository_name(ctx):
   return _join_path(ctx.attr.repository, ctx.label.package)
 
 def _impl(ctx, files=None, file_map=None, empty_files=None, directory=None,
-          entrypoint=None, cmd=None, symlinks=None):
+          entrypoint=None, cmd=None, symlinks=None, env=None, debs=None, output=None):
```

Comment:
Only output needs to be overridden here for the docker autoconfigure rule

Reply:
Done. Now only output is overridden and other changes are reverted.